examples/Request/CustomizedPayloadRequest.php

```diff
@@ -5,15 +5,14 @@
 use Fesor\RequestObject\PayloadResolver;
 use Fesor\RequestObject\RequestObject;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\Validator\Constraints as Assert;
 
 class CustomizedPayloadRequest extends RequestObject implements PayloadResolver
 {
     public function resolvePayload(Request $request)
     {
         $query = $request->query->all();
         // turn string to array of relations
-        if (isset($query['includes'])) {
```

Comment:
I don't care if it contains empty value. So in this particular case `isset` is ok.